tests/validation/cattlevalidationtest/core/test_github.py

```diff
@@ -174,14 +232,15 @@ def test_github_auth_config_valid_user(github_client, github_request_token):
 #   test that user has access
     assert schemas.status_code == 200
 
-    switch_off_auth(github_client)
 
-
-@if_github
-def test_github_auth_config_api_whitelist_users(github_client):
-
-    github_client.create_githubconfig(allowedUsers=['ranchertest01',
-                                                    'ranchertest02'])
+@pytest.mark.skipif(True, reason="Can't create githubconfigs with users o"
```

Comment:
Haha.  of all the places to split the string, you chose the middle of a word? :-P
no biggie.  just me being nitpicky